src/OpenSage.Game/Logic/Object/Draw/W3dHordeModelDraw.cs

```diff
@@ -9,6 +10,39 @@ public class W3dHordeModelDrawModuleData : W3dModelDrawModuleData
         internal static W3dHordeModelDrawModuleData Parse(IniParser parser) => parser.ParseBlock(FieldParseTable);
 
         internal static new readonly IniParseTable<W3dHordeModelDrawModuleData> FieldParseTable = W3dModelDrawModuleData.FieldParseTable
-            .Concat(new IniParseTable<W3dHordeModelDrawModuleData>());
+            .Concat(new IniParseTable<W3dHordeModelDrawModuleData>
+            {
+                { "StaticModelLODMode", (parser, x) => x.StaticModelLODMode = parser.ParseBoolean() },
+                { "LodOptions", (parser, x) => x.LODOptions.Add(LODOption.Parse(parser)) }
+            });
+
+        public bool StaticModelLODMode { get; internal set; }
+        public List<LODOption> LODOptions { get; internal set; } = new List<LODOption>();
+    }
+
+    public sealed class LODOption
```

Comment:
Could you rename this to `LodOption`, to match .NET naming convention that acronyms with three letters or more should be Pascal-cased?